### DISCUSSION TITLE
SIDKick-pico with 1541 Ultimate II+

### DISCUSSION THREAD

**Discussion by macacob** [frntc/SIDKick-pico#74] [Q&A]
Has anyone else used a SIDKick-pico with an 1541 Ultimate II+ or II+L device and encountered a problem with the configuration menu? I recently built my firs SKpico DAC 0.2 with the 0.21 firmware and am using it through PWM rather than wiring up the DAC.  When I don't have anything in the cartridge port it seems to work perfectly, but when I have the 1541 Ultimate II+ plugged in and try to access the configuration menu I instead get the "unstable bus communication detected" error screen.  I am using the older version of the 1541 Ultimate from around 2020, not the Ultimate II+L, if that makes a difference. The Ultimate II+ allows for dual SID emulation and I thought this might be the problem but I have also disabled this in settings and am still getting the error screen when I try to enter configuration.   So, my questions are:  

1.) Can anyone else with a 1541 Ultimate II+ or II+L test to see if they get the same issue. 

2.) Did I build my SIDkick poorly(definitely plausible), or might something else be causing this issue?

Documentation for the Audio settings of the Ultimate II+ can be found here: https://1541u-documentation.readthedocs.io/en/latest/quick_guide.html#audio

**Comment by frntc** [comment]
I don't have any of these cartridges, but the manual sounds as if the only passively read the bus when emulating the SID. Maybe just attaching the cartridge already has an influence on the bus signals. I assume you can modify the bus timings such that the SKpico works both with and without the carts.

**Reply by macacob** [reply]
Thanks for this.  Just to clarify. The SKpico works fine with the cartridge as far as producing sound. I have used it to play games and listen to SID Tunes from the Ultimate II+ and it sounds great.  It is just the configuration menu that seems to give a problem. (I haven't tested paddles or a mouse though, I should do that). At any rate, I can always unplug the cartridge, change configuration, and then turn off the PC and plug the cartridge back in if needed.

**Reply by frntc** [reply]
Sure, but it should not be necessary :-) (and I'd be curious if other timings remove the warning/error -- they should!)
